generators/server/templates/src/main/resources/config/_application-dev.yml

```diff
@@ -25,6 +25,8 @@ eureka:
 <%_ } _%>
 
 spring:
+    profiles:
+        active: dev,swagger
```

Comment:
I have actually tried this last time but I was getting some issues with this is that `dev` profile will be loaded first from `application.yml` in boot sequence, which would load this file. Now from this file spring would try to load the dev profile again, which would go into an infinite loop. But seems like its working fine now. So yes this could be a solution for https://github.com/jhipster/generator-jhipster/issues/4130 but then again the proper solution would be to use `spring.profiles.include` and with the fix for that [landing in SB 1.4.1](https://github.com/spring-projects/spring-boot/issues/6833) we could very well do that later. 
But I dont think this will help with https://github.com/jhipster/generator-jhipster/issues/4132 which is far serious issue than swagger not enabled when run from IDE